### ISSUE TITLE
Bells and Whistles for the All Sky Camera image.

### ISSUE THREAD

**Issue by dyerlytle** [LowellObservatory/NightWatch#16]
On the Lux.Lowell.edu web site, the all sky camera image has a few features.  There is a short animation of the last hour of images, there is a difference image, there is a color inverted image, and there is a "source detections image".  Do we need all of these features in NightWatch?

**Comment by loxlig** [comment]
Whilst checking the web server log one can easily see what page(s) are being looked at for any moment. Among the popular pages involving the allsky camera are the animation page and the generic allsky image which refreshes every 60 seconds. From the generic allsky image is a link to a slightly higher resolution image which shows some interest by a number of viewers. The negative image, the sky difference image and the source detections image are not viewed as often.

The source detections image was an experiment which uses sextractor to find star images with some tweaks to reject bogus detections. This was used initially in order to make some kind of guess at the current sky conditions based on cloud levels. This task ended up becoming more complex than initially expected so I abandoned it in favour of other projects which gave a better return on time investment.
I have since learned how to make the source detections using the astropy subpackage photutils. To me it is more straight forward and easier to use than sextractor. I may revisit the task of categorizing the sky conditions after becoming more familiar with astropy.photoutils.detection.

There was some favourable input from several viewers regarding the optional features. The colour inverted image, sky difference image, source detections image and the recently added night summary image come with relatively low overhead compared to the allsky animation which involves relatively greater overhead on system resources. The colour inverted image enhances contrast making it easier to see a feature that may, or may not, be a cloud. At the time the features were added I viewed them from the perspective of an observer and considered features I would want included as an observer.

Night Summary Image:
![night_summary](https://user-images.githubusercontent.com/40808816/50491909-fc774300-09d1-11e9-99d8-0b826d34e27b.jpg)
